CONTRIBUTING.md

```diff
@@ -14,6 +14,40 @@ Before sending your pull requests, make sure you do the following:
 -   Run the [unit tests](#running-unit-tests).
 
 ## How to become a contributor and submit your own code
+![Screen Shot 2022-08-30 at 7 27 04 PM](https://user-images.githubusercontent.com/42785357/187579207-9924eb32-da31-47bb-99f9-d8bf1aa238ad.png)
+
+### Typical Pull Request Workflow - 
+**1. New PR**
+   - As a contributor, you submit a New PR on GitHub.
+   - We inspect every incoming PR and add certain labels to the PR such as `size:`, `comp:` etc. At this stage we check if the PR is valid and meets certain quality requirements. 
```

Comment:
Generally we are not updating these for draft or WIP PRs with addition commits as  the first time the PR land in the repo is auto-classified with these labels and generally never updated.